mods/basic-themer.wh.cpp

```diff
@@ -286,7 +287,8 @@ void LoadSettings(void)
     }
 }
 
-const WindhawkUtils::SYMBOL_HOOK hooks[] = {
+// Windows 11 23H2 and before
+const WindhawkUtils::SYMBOL_HOOK uDWMDllHooks[] = {
     {
         {
             L"public: long __cdecl CWindowList::GetSyncedWindowDataByHwnd(struct HWND__ *,class CWindowData * *)"
```

Comment:
Instead of adding `uDWMDllHooks_win11`, it'd be simpler to just add the alternative symbol to the array here, like:

```suggestion
            L"public: long __cdecl CWindowList::GetSyncedWindowDataByHwnd(struct HWND__ *,class CWindowData * *)",
            L"public: void __cdecl CWindowList::GetSyncedWindowDataByHwnd(struct HWND__ *,class CWindowData * *)"
```

More importantly, having two `HookSymbols` calls for the same module will screw up the caching. It'd be fine if you checked the Windows/module version in runtime and called only right one, but you're always calling both.